src/spetlr/power_bi/SparkPandasDataFrame.py

```diff
@@ -0,0 +1,314 @@
+from datetime import datetime
+from typing import Callable, Dict, List, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from dateutil.parser import parse
+from pandas.core.generic import NDFrameT
+from pyspark.sql import DataFrame
+from pytz import timezone, utc
+
+from spetlr.exceptions import SpetlrException
+from spetlr.spark import Spark
+
+
+class SparkPandasDataFrame:
```

Comment:
Isnt there a .ToPandas() function we can use instead of this custom class?

Reply:
https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/pandas_pyspark.html

Reply:
Thanks for the suggestion, but this class serves many different purposes. 
1) It solves many problems Pandas create when dealing with types, e.g. Pandas treat null values as float.NaN, which is very weird. 
2) Pandas are not good in parsing a timestamp column where one row has milliseconds, and the other does not. It casts an exception. 
3) I wanted to make a flexible logic where I can implement a new REST API request with almost no code (only the schema with possible transformations).
4) I wanted to make it possible to use the logic without Spark. Everything in the PowerBi class works without Spark (Pandas is just a pypi library) except of the get Spark dataframe methods. :)